ansible/roles/open-env-gcp-remove-project/tasks/main.yml

```diff
@@ -0,0 +1,89 @@
+---
+- name: Get project name
+  set_fact:
+    project_name: "openenv-{{ guid }}-{{ requester_name }}"
+
+- name: Call GCP get token role
+  include_role:
+    name: gcp-get-token
+
+- name: Get project
+  uri:
+    url: "https://cloudresourcemanager.googleapis.com/v3/projects/{{ project_name }}"
+    method: GET
+    headers:
+      Authorization: "Bearer {{ auth_response.json.access_token }}"
+      Accept: "application/json"
+      Content-Type: "application/json"
+    body:
```

Comment:
Stray `body:`?

Reply:
the instructions say to send {} in json.. so i was thinking this was necessary? I am happy to remove it.